nautobot/dcim/views.py

```diff
@@ -211,25 +211,9 @@ class SiteView(generic.ObjectView):
     queryset = Site.objects.select_related("region", "tenant__tenant_group")
 
     def get_extra_context(self, request, instance):
-        stats = {
-            "rack_count": Rack.objects.restrict(request.user, "view").filter(site=instance).count(),
-            "device_count": Device.objects.restrict(request.user, "view").filter(site=instance).count(),
-            "prefix_count": Prefix.objects.restrict(request.user, "view").filter(site=instance).count(),
-            "vlan_count": VLAN.objects.restrict(request.user, "view").filter(site=instance).count(),
-            "circuit_count": Circuit.objects.restrict(request.user, "view")
-            .filter(circuit_terminations__site=instance)
-            .count(),
-            "vm_count": VirtualMachine.objects.restrict(request.user, "view").filter(cluster__site=instance).count(),
-        }
-        rack_groups = (
-            RackGroup.objects.annotate(rack_count=count_related(Rack, "group")).restrict(request.user, "view")
-            # .filter(site=instance)
-        )
-        locations = (
-            Location.objects.restrict(request.user, "view")
-            .filter(site=instance)
-            .select_related("parent", "location_type")
-        )
+        stats = {}
+        rack_groups = ()
+        locations = ()
```

Comment:
It seems like this might need to stay? Or are these counts being tabulated elsewhere now?

Reply:
There are similar stats and tables on `LocationListView`. Since Site model technically cannot query for those stats anymore.

Reply:
I am dumb, because Site is going away. Genius.